sub-projects/device/utils/compare.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+import json
+from PIL import Image
+
+class Rekgoniton(object):
+	"""
+	A class that creates a client object for access to AWS Rekgoniton.
+	"""
+	def __init__(self):
+		self.client = boto3.client("rekognition")
+		self.image_list = ['cameron.jpg','rachel.jpg'];
```

Comment:
What is this for? Why is it hard-coded?

Reply:
I am guessing we have to specifically tell the AWS Rekognition which file to compare the image to?

Reply:
Is there any way we can get all of the files we have in the S3 and then just loop through that way, instead of hard-coding? This is going to cause problems.